### DISCUSSION TITLE
Product recommendation JS error without using merge JS

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#293] [5. General]
Hi @vovayatsyuk 

We noticed product recommendation is not working correctly without using JS merge, after upgrading Magento (2.4.6-p6) and Breeze (latest), works fine if we enable JS merge.

**Comment by vovayatsyuk** [comment]
Thanks for reporting! Fixed in 1.0.4.

https://github.com/breezefront/module-breeze-magento-enterprise/commit/a0918bc9786b72ba2f4f0c3a3fc1dba3f4048596
